CHANGELOG.md

```diff
@@ -347,6 +347,18 @@ Library management
 
   Previously such `.agda-lib` files were ignored.
 
+* Now supports reading files with extension `.lagda.typ`, and use the parser for
+  markdown files to parse them.
+  To edit such files in Emacs with Agda support, one needs to add the line
+  ```elisp
+  (add-to-list 'auto-mode-alist '("\\.lagda.md\\'" . agda2-mode))
+  ```
+  to `.emacs`.
+
+  Code generation for Typst is unsupported. One may generate HTML with typst input,
```

Comment:
I don't understand what is meant by "code generation for Typst is unsupported".

Reply:
I added a bit of explanation. Do you understand it now?